mandatory/debugging-practice/script.js

```diff
@@ -89,13 +89,14 @@ function render() {
     });
 
     //add delete button to every row and render again
-    let delButton = document.createElement("button");
+    let delBut = document.createElement("button");
     delBut.id = i + 5;
     cell5.appendChild(delBut);
     delBut.className = "btn btn-warning";
     delBut.innerHTML = "Delete";
-    delBut.addEventListener("clicks", function () {
+    delBut.addEventListener("click", () => {
```

Comment:
Looks good to me Mahri :) I believe there are more bugs to be fixed in this task aside from the ones you have done.

Reply:
I will take a look again and also review it, I checked it quickly if you have any options so please share!

Reply:
thank you!